### PR TITLE
saga-9.726/feature-filter-icon

### PR DESCRIPTION
**Pull request by matiasmarquez**
Added filter icon and extracted SVG from [https://fonts.google.com/icons](url)

There is a short demo on Asana task. 
https://asanal.ink/21/726

### CODE REVIEW

**Comment by matiasmarquez** [comment]
@rmcarias Maybe this is more accurated at what we are looking for ?

image

File: src/shared/images/icons/filter.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/></svg>
```
Comment (rmcarias) [inline src/shared/images/icons/filter.svg:1]: newline

**Review by rmcarias — CHANGES_REQUESTED** [review]

File: src/shared/images/icons/filter.svg (hunk 1 above)
Comment (matiasmarquez) [reply src/shared/images/icons/filter.svg]: Done

**Review by rmcarias — APPROVED** [review]

**Review by rmcarias — APPROVED** [review]

**Pull request merged**